ocs_ci/ocs/resources/ocs.py

```diff
@@ -212,6 +212,12 @@ def get_ocs_csv():
         CSVNotFound: In case no CSV found.
 
     """
+
+    odf_csv = OCP(kind="operator", resource_name="odf-operator.openshift-storage")
```

Comment:
I think we should make this name of operator based on the version.
So if version < 4.9 use 
OCS_OPERATOR_NAME from here
https://github.com/red-hat-storage/ocs-ci/blob/baeb91c20ca96fcb5f3d97350183b08ebec481a5/ocs_ci/ocs/defaults.py#L38-L39 .

Otherwise:
ODF_OPERATOR_NAME .

For version, please use this function

 https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/version.py#L49 

Comparing with e.g. version.VERSION_4_9

Reply:
You are relaying on ocs/odf version installed for the check which can be that ocs/odf is not installed at all (The case of the PR).
What we can do is check for both of the names and if both check fails then return csv not found. WDYT?